### DISCUSSION TITLE
How do a get LazyVim to show my JSON file as text?

### DISCUSSION THREAD

**Discussion by dannyfritz** [LazyVim/LazyVim#1486] [Q&A] [answered]
Right now, LazyVim by default only shows the current line as text, and the other lines as formatted text. How do I turn this off? I can see how this might be a useful feature, but for some reason it quite upsets me. I'm pretty new to nvim and don't really know where to look or what this feature is called.

[Screencast from 2023-09-24 12-09-17.webm](https://github.com/LazyVim/LazyVim/assets/315788/8e254bb1-dc8d-43b2-81ae-d1620a42bffd)

**Comment by dpetka2001** [comment, marked as answer]
In your `/lua/config/options.lua` file put
```lua
vim.opt.conceallevel = 0
```
Try to see if this is what you want to achieve.

**Reply by dannyfritz** [reply]
Yes, thank you. Worked like a charm.

https://neovim.io/doc/user/options.html#%27conceallevel%27
